pom.xml

```diff
@@ -1,151 +1,209 @@
+            <!-- WorldEdit, WorldGuard -->
+            <id>sk89q-repo</id>
+            <url>https://maven.enginehub.org/repo/</url>
+        </repository>
+        <repository>
+            <!-- Various -->
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+        <repository>
+            <!-- Multiverse -->
+            <id>multiverse-repo</id>
+            <url>https://repo.onarandombox.com/content/repositories/multiverse-snapshots/</url>
+        </repository>
+        <repository>
+            <!-- BlockLocker -->
+            <id>codemc-repo</id>
+            <url>https://repo.codemc.org/repository/maven-public/</url>
+        </repository>
+        <repository>
+            <!-- LWCX (Lightweight Chests Extended) -->
+            <id>ender-zone-repo</id>
+            <url>https://ci.ender.zone/plugin/repository/everything/</url>
+        </repository>
+        <repository>
+            <!-- ProtocolLib -->
+            <id>dmulloy2-repo</id>
+            <url>https://repo.dmulloy2.net/repository/public/</url>
+        </repository>
+        <repository>
+            <!-- LibsDisguises -->
+            <id>md_5-public</id>
+            <url>https://repo.md-5.net/content/repositories/public/</url>
+        </repository>
+        <repository>
+            <!-- EssentialsX releases -->
+            <id>essentialsx-repo</id>
+            <url>https://repo.essentialsx.net/releases/</url>
+        </repository>
+        <repository>
+            <!-- EssentialsX snapshots -->
+            <id>essentialsx-snapshot-repo</id>
+            <url>https://repo.essentialsx.net/snapshots/</url>
+        </repository>
+        <repository>
+            <!-- RedProtect -->
+            <id>redprotect-repo</id>
+            <url>https://raw.githubusercontent.com/FabioZumbi12/RedProtect/mvn-repo/</url>
+        </repository>
+        <repository>
+            <!-- CoreProtect -->
+            <id>coreprotect-repo</id>
+            <url>https://maven.playpro.com</url>
+        </repository>
+        <repository>
+            <!-- Dynmap -->
+            <id>dynmap-repo</id>
+            <url>https://repo.mikeprimm.com/</url>
         </repository>
     </repositories>
     <pluginRepositories>
         <pluginRepository>
+            <!-- Apache Snapshots -->
             <id>apache.snapshots</id>
             <url>https://repository.apache.org/snapshots/</url>
         </pluginRepository>
     </pluginRepositories>
-    <scm>
-        <connection>scm:git:ssh://github.com/eccentricdevotion/TARDIS</connection>
-        <developerConnection>scm:git:ssh://github.com/eccentricdevotion/TARDIS</developerConnection>
-    </scm>
-    <ciManagement>
-        <system>jenkins</system>
-        <url>http://tardisjenkins.duckdns.org:8080</url>
-    </ciManagement>
-    <!-- Profiles are used to detect whether this is a local or Jenkins build and adjust the build number accordingly -->
```

Comment:
I want to keep the Jenkins build number as part of the plugin.yml version. Your changes remove this.
These profiles give Jenkins builds a different version string than local builds:
Jenkins -> 4.7.3-b2333
Local -> 4.7.3-dd.MM.yy-HH:mm
This makes it much easier to deal with issues, as we know which build the server might be using.

Reply:
Unrelated, but having the Local be a git hash would be a lot easier to deduce. Future?

Reply:
Yeah, maybe, but I often have multiple test jars during testing before I've actually made a commit.